software/src/modules/ac011k/ac011k.cpp

```diff
@@ -1493,8 +1446,37 @@ void AC011K::loop()
                 }
                 break;
 
+            case 0x0C:
+                /*
+                              0  1  2  3  4  5  6  7  8  9  0  10 11 12 13 14 
+                         000: FA 03 00 00 0C 02 05 00 11 09 01 00 00 73 79            CtrlGetS20OpenStop -> Off
```

Comment:
for all 0C cases, the data field is like this:
4    5    6             7              8      9              10              11                 12 13 14 15 .....                           xxxxx
0C ss  reclen_lo reclen_hi type subcmd  datalen_lo  datalen_hi (data in case data_len is >0) .... crc